specs/book.toml

```diff
@@ -2,7 +2,6 @@
 authors = ["Celestia Labs"]
 language = "en"
 multilingual = false
-src = "src"
```

Comment:
This config wasn't doing anything because "src" is the default

> src: By default, the source directory is found in the directory named src directly under the root folder. But this is configurable with the src key in the configuration file.

Ref: https://rust-lang.github.io/mdBook/format/configuration/general.html#general-metadata